### PR TITLE
Add a License

### PR DESCRIPTION
**Pull request by Crocmagnon**
## Add a license
Please consider adding a license to this project so we can use it ! (or not)

### Why
By the wording on your developer website ([here](https://developer.spotify.com/web-api/tutorial/)), I presume you want to allow developers to use it freely in their projects. If that's the case, I suggest you to use the MIT License (included in this pull request).

### What if you don't
If no license is provided, it means this is under copyright (see http://choosealicense.com/no-license/). Maybe that's what you want, but in this case please be explicit and add a line in a LICENSE or README file 😄 

### More
You can find more about licenses (in plain english) here : http://choosealicense.com/licenses/

### CODE REVIEW

**Comment by JMPerez** [comment]
I have just added the Apache License we use in the rest of open source projects. See https://github.com/spotify/web-api-auth-examples/commit/c76a864847b2bf24daf335908bce593191cb4441.

Thanks for the feedback!

**Comment by Crocmagnon** [comment]
Thank you for the clarification :)
